contents/blog/best-hipaa-compliant-ab-testing-tools.md

```diff
@@ -7,7 +7,7 @@ showTitle: true
 hideAnchor: true
 author:
   - andy-vandervell
-featuredImage: ../images/blog/hipaa-compliant-ab-testing/hipaa.jpeg
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
```

Comment:
You don't want to use the hipaa hippo? I don't feel strongly.

Reply:
I updated two at the same time, so it was appearing twice on the blog page.